src/app/codegen-data-model/CodegenDataModel.cpp

```diff
@@ -232,10 +233,17 @@ bool CodegenDataModel::EmberCommandListIterator::Exists(const CommandId * list,
 }
 
 CHIP_ERROR CodegenDataModel::Invoke(const InteractionModel::InvokeRequest & request, TLV::TLVReader & input_arguments,
-                                    InteractionModel::InvokeReply & reply)
+                                    CommandHandler * handler)
 {
-    // TODO: this needs an implementation
-    return CHIP_ERROR_NOT_IMPLEMENTED;
+    // TODO: CommandHandlerInterface support is currently
```

Comment:
nit, do we have an issue tracker that we can link to. Or is your plan to do this in a follow up really soon?